modules/dhcp/providers/server/isc.rb

```diff
@@ -188,6 +188,7 @@ def omcmd cmd, msg=nil
         @om = IO.popen("/bin/sh -c '#{om_binary} 2>&1'", "r+")
         @om.puts "key #{Proxy::DhcpPlugin.settings.dhcp_key_name} \"#{Proxy::DhcpPlugin.settings.dhcp_key_secret}\"" if Proxy::DhcpPlugin.settings.dhcp_key_name && Proxy::DhcpPlugin.settings.dhcp_key_secret
         @om.puts "server #{name}"
+        @om.puts "port #{Proxy::DhcpPlugin.settings.dhcp_omapi_port} \"#{Proxy::DhcpPlugin.settings.dhcp_omapi_port}\"" if Proxy::DhcpPlugin.settings.dhcp_omapi_port
```

Comment:
Port value is being appended twice. This is probably not intentional? Might be worth adding a default value for the port (in which case the conditional check can go).